### PR TITLE
rename target to hosting

### PR DESCRIPTION
**Pull request by ianzhang366**
Signed-off-by: Ian Zhang <izhang@redhat.com>

CRD update is at:
https://github.com/stolostron/backplane-operator/pull/170

### CODE REVIEW

File: api/v1alpha1/hypershiftdeployment_types.go [hunk 1]
```diff
@@ -98,18 +98,18 @@ type HypershiftDeploymentSpec struct {
...
 	//if not provided, the default is "clusters"
 	// +optional
-	TargetNamespace string `json:"targetNamespace"`
+	HostingNamespace string `json:"hostingNamespace"`
 
-	//TargetManagedCluster only applies to ManifestWork, and specifies which managedCluster's namespace the manifestwork will be applied to.
+	//HostingManagedCluster only applies to ManifestWork, and specifies which managedCluster's namespace the manifestwork will be applied to.
 	//If not specified, the controller will flag an error condition.
-	//The TargetManagedCluster would be the management cluster of the hostedcluster and nodepool generated
+	//The HostingManagedCluster would be the management cluster of the hostedcluster and nodepool generated
```
Comment (zhujian7) [inline api/v1alpha1/hypershiftdeployment_types.go]: This should be `HostingCluster`？

**Comment by zhujian7** [comment]
remember to update [docs](https://github.com/stolostron/hypershift-deployment-controller/tree/main/docs).

File: api/v1alpha1/hypershiftdeployment_types.go (hunk 1 above)
Comment (ianzhang366) [reply api/v1alpha1/hypershiftdeployment_types.go]: https://coreos.slack.com/archives/C02V40F8BH7/p1648658544845079

overhere it didn't specify that...

Comment (zhujian7) [reply api/v1alpha1/hypershiftdeployment_types.go]: see https://coreos.slack.com/archives/C02V40F8BH7/p1648526843528209

Comment (ianzhang366) [reply api/v1alpha1/hypershiftdeployment_types.go]: done

**Comment by zhujian7** [comment]
@ianzhang366 CI failed, please check.

**Comment by ianzhang366** [comment]
@zhujian7

**Pull request merged**